docs/MO_DG/prepare_model/MO_Python_API.md

```diff
@@ -116,8 +123,9 @@ Example of using the ``LayoutMap`` class to change the layout of a model input:
 .. code-block:: py
    :force:
 
-   from openvino.tools.mo import convert_model, LayoutMap
-   
+   import openvino as ov
+   from openvino.tools.mo import LayoutMap
```

Comment:
Same here.

Reply:
No need to change it. 
@sgolebiewski-intel please revert this change.

Reply:
I have reverted it in the https://github.com/openvinotoolkit/openvino/pull/19084/commits/24d1321aa23402e051f7925957e6eedaacd7d9ff commit.